<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Zhmot</author>
 <class>MainSV</class>
 <widget class="QWidget" name="MainSV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>519</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>741</width>
    <height>519</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>741</width>
    <height>519</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Что cегодня на seasonvar.ru</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>seasonvar.ico</normaloff>seasonvar.ico</iconset>
  </property>
  <widget class="QLabel" name="DescriptionSeries">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>681</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Описание сериала</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ImageSeries">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>35</y>
     <width>200</width>
     <height>281</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Постер сериала</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="TableSeries">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Наименование сериала</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="UpdateButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayerButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>480</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>play.ico</normaloff>play.ico</iconset>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>TableSeries</tabstop>
  <tabstop>UpdateButton</tabstop>
  <tabstop>PlayerButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
